.gitmodules

```diff
@@ -1,6 +1,7 @@
 [submodule "third_party/triton"]
 	path = third_party/triton
 	url = https://github.com/ROCmSoftwarePlatform/triton.git
+	branch = aotriton-hyperjump
```

Comment:
What are the new changes in Triton?  Can we look at getting them upstream or do these need to remain as a forked enhancement?

Reply:
Mainly removal of NVIDIA dependencies, improvement of their build system, and https://github.com/triton-lang/triton/pull/4255
Will likely be a forked enhancement for some more time.